### PR TITLE
net-ftp/uftpd: bump to version 2.5

### PR DESCRIPTION
**Pull request by oz123**
Package-Manager: Portage-2.3.40, Repoman-2.3.9

### CODE REVIEW

File: net-ftp/uftpd/uftpd-2.5.ebuild [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```
Comment (Zlogene) [inline net-ftp/uftpd/uftpd-2.5.ebuild]: why not 7?

Comment (oz123) [reply net-ftp/uftpd/uftpd-2.5.ebuild]: no reason. Changing it now.

**Comment by oz123** [comment]
@Zlogene done. Now on EAPI 7.

File: net-ftp/uftpd/uftpd-2.5.ebuild [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+HOMEPAGE="https://github.com/troglobit/uftpd"
+SRC_URI="https://github.com/troglobit/${PN}/releases/download/v${PV}/${P}.tar.xz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+RDEPEND="
+		!net-misc/uftp
```
Comment (mgorny) [inline net-ftp/uftpd/uftpd-2.5.ebuild]: Please use single tab for indent.

File: net-ftp/uftpd/uftpd-2.5.ebuild [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+		!net-ftp/atftp"
+
+DEPEND="
+	dev-libs/libite
+	dev-libs/libuev"
+
+RDEPEND="${DEPEND}"
```
Comment (mgorny) [inline net-ftp/uftpd/uftpd-2.5.ebuild]: Ok, this makes no sense. You've setting `RDEPEND` first, then overwriting it here.

**Comment by oz123** [comment]
@mgorny ,
You are correct, I was overriding my definition. This probably a result of some earlier git merge I did.

**Comment by mgorny** [comment]
Ok, it's almost good, except it installs:

```
./usr/share/doc/uftpd-2.5/LICENSE.lz
```

In Gentoo we don't install licenses because they are used directly by ebuilds. I'm sorry we haven't spotted this earlier. I think you may try overriding `doc_DATA` on emake command-line. Or just removing the file after install call.

**Comment by oz123** [comment]
@mgorny done. I find a patch overriding doc_DATA a bit nicer then installing and then removing.

File: net-ftp/uftpd/files/0001-remove-license.patch [hunk 4]
```diff
@@ -0,0 +1,11 @@
+--- uftpd-2.5/Makefile.am	2018-06-06 19:32:37.000000000 +0200
++++ uftpd-2.5/Makefile.am.1	2018-06-11 09:07:10.227975263 +0200
+@@ -1,6 +1,6 @@
+ SUBDIRS            = src man
+-doc_DATA           = README.md LICENSE
+-EXTRA_DIST         = README.md LICENSE ChangeLog.md
++doc_DATA           = README.md
++EXTRA_DIST         = README.md ChangeLog.md
```
Comment (mgorny) [inline net-ftp/uftpd/files/0001-remove-license.patch]: Actually, modifying `EXTRA_DIST` incorrect. Not that it matters for Gentoo, so I won't block you on this.

**Comment by oz123** [comment]
@mgorny I changed the patch to only modify `doc_DATA`. I believe the ebuild can now be merged into the master.

**Comment by mgorny** [comment]
Yes, I think it would work, and if it worked, it'd be correct. I'm leaving the choice to you; both methods have their advantages.

**Comment by oz123** [comment]
@mgorny you method works. I just needed a refresh on src_install ...
Feel free to merge if no other issues are found.

**Comment by mgorny** [comment]
Please always use tab-indent. That said, I think everything else is fine, so I'll just fix that, test it and merge it.

**Pull request closed without merging**